app/src/main/java/com/github/h3lp3rs/h3lp/CprRateActivity.kt

```diff
@@ -45,18 +50,36 @@ class CprRateActivity : AppCompatActivity() {
 
         startButton.setOnClickListener {
             if (animationIsPlaying) {
-                //heartIcon.clearAnimation()
-                startButton.text = getString(R.string.cpr_rate_button_start)
-                beepSound.pause()
-                beepSound.seekTo(0)
-                animationIsPlaying = false
+                stop()
             } else {
-                //heartIcon.startAnimation(heartBeatAnimation)
-                startButton.text = getString(R.string.cpr_rate_button_stop)
-                beepSound.start()
-                animationIsPlaying = true
+                start()
             }
 
         }
     }
+
+    /**
+     * Stop all the animations when the user does no longer see the activity
+     */
+    override fun onStop() {
+        stop()
+        super.onStop()
+    }
```

Comment:
Nice Fix for this bug

Maybe add a test to be sur the song stop when app isStopped(). This will improve our coverage and be sure this never reappeared